index.html

```diff
@@ -0,0 +1,260 @@
+            <h2>Grain+Glory</h2>
+          </div>
+          <div class="btn">
+            <button type="button" class="ruby">Ruby on rails</button>
+            <button type="button" class="css">css</button>
+            <button type="button" class="js">Javascript</button>
+            <button type="button" class="html">html</button>
+          </div>
+          <button type="button" class="btn-project">See Project</button>
+        </div>
+      </div>
+
+      <div class="card">
+        <div class="img_container"></div>
+        <div class="bottom">
+          <div class="intro">
+            <h2>Mult-Post Stories</h2>
+            <h2>Grain+Glory</h2>
+          </div>
+          <div class="btn">
+            <button type="button" class="ruby">Ruby on rails</button>
+            <button type="button" class="css">css</button>
+            <button type="button" class="js">Javascript</button>
+            <button type="button" class="html">html</button>
+          </div>
+          <button type="button" class="btn-project">See Project</button>
+        </div>
+      </div>
+
+      <div class="card">
+        <div class="img_container"></div>
+        <div class="bottom">
+          <div class="intro">
+            <h2>Mult-Post Stories</h2>
+            <h2>Grain+Glory</h2>
+          </div>
+          <div class="btn">
+            <button type="button" class="ruby">Ruby on rails</button>
+            <button type="button" class="css">css</button>
+            <button type="button" class="js">Javascript</button>
+            <button type="button" class="html">html</button>
+          </div>
+          <button type="button" class="btn-project">See Project</button>
+        </div>
+      </div>
+      <div class="card">
+        <div class="img_container"></div>
+        <div class="bottom">
+          <div class="intro">
+            <h2>Mult-Post Stories</h2>
+            <h2>Grain+Glory</h2>
+          </div>
+          <div class="btn">
+            <button type="button" class="ruby">Ruby on rails</button>
+            <button type="button" class="css">css</button>
+            <button type="button" class="js">Javascript</button>
+            <button type="button" class="html">html</button>
+          </div>
+          <button type="button" class="btn-project">See Project</button>
+        </div>
+      </div>
+      <div class="card">
+        <div class="img_container"></div>
+        <div class="bottom">
+          <div class="intro">
+            <h2>Mult-Post Stories</h2>
+            <h2>Grain+Glory</h2>
+          </div>
+          <div class="btn">
+            <button type="button" class="ruby">Ruby on rails</button>
+            <button type="button" class="css">css</button>
+            <button type="button" class="js">Javascript</button>
+            <button type="button" class="html">html</button>
+          </div>
+          <button type="button" class="btn-project">See Project</button>
```

Comment:
- [ ]  line 95,this div should be a ul instead and the buttons should be inside a list tag li
just like you did on line 166